lib/apiservers/portlayer/restapi/handlers/containers_handlers.go

```diff
@@ -255,6 +257,13 @@ func (handler *ContainersHandlersImpl) RemoveContainerHandler(params containers.
 		case exec.RemovePowerError:
 			return containers.NewContainerRemoveConflict().WithPayload(&models.Error{Message: err.Error()})
 		default:
+			f, ok := err.(types.HasFault)
+			if ok {
+				switch f.Fault().(type) {
+				case *types.HostNotConnected:
+					return containers.NewContainerRemoveDefault(501).WithPayload(&models.Error{Message: "Couldn't remove container because the host the container is placed on is not connected"})
```

Comment:
Let's shorten this message and remind the user to try again later.

```
Couldn't remove container.  The ESX host is temporarily disconnected.  Please try again later.
```

Reply:
Roger